lib/whitehall/rummageable.rb

```diff
@@ -92,7 +92,7 @@ def make_request(method, *args)
         response = nil
         log_request(method, *args)
         call_time = Benchmark.realtime do
-          response = RestClient.send(method, *args, content_type: :json, accept: :json)
+          response = RestClient.send(method, *args, content_type: :json, accept: :json, authorization: "Bearer #{ENV['RUMMAGER_BEARER_TOKEN'] || 'example'}")
```

Comment:
might be slightly cleaner to do: `ENV.fetch('RUMMAGER_BEARER_TOKEN', 'example')`

Reply:
True, I think this was following the existing pattern elsewhere.